webform_civicrm_file_case_emails.module

```diff
@@ -9,4 +9,17 @@
  * Implements hook_mail_alter().
  */
 function webform_civicrm_file_case_emails_mail_alter(array &$message) {
+  // Process webform submission emails only.
+  if ($message['module'] !== 'webform' || $message['key'] !== 'submission') {
```

Comment:
This check is already been duplicated in the `isMessageValid` function of the `WcfceMailProcessor` class. Can we move the logic to just one place for the checks.